install_config/aggregate_logging.adoc

```diff
@@ -937,6 +962,65 @@ existing `journald` log driver configuration. It is recommended to use the
 Logging Driver] for instructions to change your existing log driver
 configuration to `json-file`.
 
+[[fluentd-file]]
+*Configuring Fluentd to Send Logs to a File*
+
+Fluentd writes logs to a file you specify with the `LOGGING_FILE_PATH` environment variable, by default `/var/log/fluentd/fluentd.log`.  
+When the current Fluentd log file reaches a specified size, {product-title} automatically renames the *fluentd.log* log file so that new logging data can be collected.
+You can control how many of the renamed files that {product-title} retains. 
+
+The following example shows logs in a cluster where the maximum log size is 1Mb and four logs should be retained. When the *fluentd.log* reaches 1Mb, {product-title} deletes the current *fluentd.log.4*, renames the each of the Fluentd in turn, and creates a new *fluentd.log*.  
+
+----
+fluentd.log     0b
+fluentd.log.1  1Mb
+fluentd.log.2  1Mb
+fluentd.log.3  1Mb
+fluentd.log.4  1Mb
+----
+
+Use the *logs* utility to print out the contents of Fluentd log files. For example:
+
+----
+oc exec <pod> logs <1>
+----
+
+<1> Specify the name of the Fluentd or Elasticsearch pod.
+
+For example:
+
+----
+oc exec logging-es-data-master-2-deploy logs
+oc exec logging-kibana-1 logs
+----
```

Comment:
These changes are in fluentd pod, not the kibana or elasticsearch, so the command should be "oc exec $fluentd-pod -- logs"

Reply:
@VladMasarik @QiaolingTang Do we need the dashes before `logs`? There has been some conflicting information.

Reply:
@mburke5678 It doesn't matter. I tried "oc exec logging-fluentd-lmvms logs" and "oc exec logging-fluentd-lmvms -- logs", both of them are valid

Reply:
I have just never seen anyone use the `--`. And since I had no way to test it at that time, I just didn't want to risk showing falsie information.

Reply:
The command in the docs has no `--`. 
`oc exec logging-fluentd-lmvms logs`